app/controllers/reliefs/ReliefDeclarationController.scala

```diff
@@ -64,6 +64,7 @@ class ReliefDeclarationController @Inject()(mcc: MessagesControllerComponents,
               Future.successful(BadRequest(views.html.global_error("ated.client-problem.title",
                 "ated.client-problem.header", "ated.client-problem.message",
                 Some(appConfig.agentRedirectedToMandate), Some("ated.client-problem.HrefMessage"), None, appConfig)))
+            case NOT_FOUND => Future.successful(Redirect(controllers.reliefs.routes.ReliefsSentController.view(periodKey)))
```

Comment:
Currently works as it is. However, not sure if it is good practice for the "NOT_FOUND" case to be the same as the "OK" case

Reply:
Testing manually locally and throws the WARN as required without throwing technical difficulties warning for submission of reliefs with users sharing same credentials. Functionally works - and its handled by the reliefSentController so can see no reason that this is not valid for the scenario identified